README.md

```diff
@@ -27,22 +27,8 @@ Kata Containers is working to build a global, diverse and collaborative communit
 
 ## Users
 
-Since Kata Containers does not yet provide an installation option, the current
-advice for users is to install either Clear Containers or runV since both
-projects will provide a migration path to Kata Containers at a later date.
-
-If you do not already have an installation of either project, Clear Containers
-may be the simplest option as packages for common Linux\* distributions are
-provided. However, your choice may depend on the particular project features
-that interest you.
-
-### Installing Clear Containers
-
-See https://github.com/clearcontainers/runtime/wiki/Installation.
-
-### Installing runV
-
-See https://github.com/hyperhq/runv.
+See [Kata Containers installation user guides](https://github.com/kata-containers/documentation/blob/master/install/README.md#kata-containers-installation-user-guides) for details on how to install Kata Containers for your preferred 
```

Comment:
Hi @nitkon - thanks for raising.

Please can you update the table of contents links at the top of the doc as they still reference the old section names.

@klynnrif - ptal.

Reply:
@jodh-intel @klynnrif Done!

Reply:
Thanks - I'd be tempted to remove the `#kata-containers-installation-user-guides` as that's a link to the top of the document anyway.

Reply:
@jodh-intel : Fixed.  Thanks!